app/lib/setUpRendererAndDrawChords.ts

```diff
@@ -1,11 +1,11 @@
 import VexFlow from "vexflow";
```

Comment:
This is partly my fault too, but I noticed it might be good to have a unified way of importing vexflow. Right now I'm seeing the following throughout the codebase:
			- `import * as Flow from "vexflow";`
			- `import VexFlow from "vexflow";`
			- `import { Renderer } from "vexflow";`
			- and the combo:
				- `import VexFlow from "vexflow";`
				- `const { Renderer } = VexFlow.Flow;`